src/containers/guest-home-page/signup-form/SignupForm.js

```diff
@@ -131,6 +136,7 @@ const SignupForm = ({ handleSubmit, handleChange, handleBlur, data, errors, clos
       <Box sx={ styles.checkboxContainer }>
         <FormControlLabel
           control={ <Checkbox /> }
+          disabled={ isValid ? false : true }
```

Comment:
u can use just isValid if this is boolean, or Boolean(isValid), ternary is not needed

Reply:
fixed